data/90-crash-metrics.conf

```diff
@@ -0,0 +1 @@
+kernel.core_pattern=|/usr/bin/eos-crash-metrics %E %s %t
```

Comment:
On my system the "eos-metrics-instrumentation" ends up in "/usr/lib/x86_64-linux-gnu/eos-metrics-instrumentation". I would call the file "90-eos-crash-metrics.conf"

Reply:
Yeah, this should be a `.in` file and use `$(libexecdir)/eos-crash-metrics` which will be substituted with the right value at build time.